BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -674,9 +688,7 @@ namespace big
 							selected_color = color;
 							pearlescent = color;
 
-							g_fiber_pool->queue_job([] {
-								VEHICLE::SET_VEHICLE_EXTRA_COLOURS(player_vehicle, pearlescent, wheel_color);
-							});
+							VEHICLE::SET_VEHICLE_EXTRA_COLOURS(player_vehicle, pearlescent, wheel_color);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.